model/TaskLogBroker/RdsTaskLogBroker.php

```diff
@@ -314,83 +314,61 @@ public function getStats(TaskLogFilter $filter)
      */
     public function archive(TaskLogEntity $entity)
     {
-        $this->getPersistence()->getPlatform()->beginTransaction();
+        $qb = $this->getQueryBuilder()
```

Comment:
why don't we create an extension of this object that is related to spanner,  i don't see the benefit in this development

Reply:
Here I'm just removing the unnecessary transaction. It's only in this PR because it removes one usage of the transaction system. If you take the changes of successive commits, everything is clearer.